server.js

```diff
@@ -159,8 +158,8 @@ function Weather(weatherObject) {
 
 function Park(parkObject) {
   this.name = parkObject.name;
-  this.address = parkObject.address;
-  this.fee = parkObject.fee;
+  this.address = parkObject.addresses[0];
+  this.fee = parkObject.entrancefee[0].cost;
   this.description = parkObject.description;
   this.url = parkObject.url;
 }
```

Comment:
client.connect()
  .then(() => {
    app.listen(PORT, () => console.log(`running on ${PORT}`));
})
.catch(error => console.error(error));

// pairing together so that app won't run unless client.connect is active.